### ISSUE TITLE
Show indication of refreshing when using the load latest button (or pressing the tab button to scroll to top)

### ISSUE THREAD

**Issue by haileyok** [bluesky-social/social-app#2260]
**Is your feature request related to a problem? Please describe.**

Currently, after pressing the `LoadLatestBtn` (or pressing the home tab button), the feed scrolls to the top and refreshes. However, there is no indication that a refresh is taking place or has been completed.

**Describe the solution you'd like**

At a quick glance, `isPTRing` is set to true whenever you pull to refresh, however that does not happen when using the button to scroll up and refresh the feed. 

`RefreshControl` should have `refreshing` set to true during any refresh event, not just on pull to refresh.

~~After a quick test, it looks like using the `isRefetching` value from the query provides the desired result. Seemingly if you do this there is no need for the `isPTRing` state value, but that's just a brief test.~~

Using `isRefetching` doesn't seem to actually do what we want, I assume because of the async nature of the query and that value isn't updated quite fast enough. Get some glitchy behavior about half the time if we use that.

https://github.com/bluesky-social/social-app/assets/153161762/b5dd93b6-3ed4-48c1-8685-a43fa9b90268

**Comment by pfrazee** [comment]
Yeah that glitchy behavior is the issue. If the refresh is too fast, it looks pretty bad. Worth looking at because I hate the lack of UI feedback but not prio right now
